### PR TITLE
Added slides for 15 talks

### PR DESCRIPTION
**Pull request by SaraDGore**

### CODE REVIEW

File: _schedule/talks/2017-08-15-17-00-t1-django-vs-flask.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ presenters:
   website: https://www.davidbaumgold.com/
 room: 'Ballroom A'
 sitemap: true
-slides_url: ''
+slides_url: 'https://t.co/NiTcld5VGV'
```
Comment (FlipperPA) [inline _schedule/talks/2017-08-15-17-00-t1-django-vs-flask.md]: Should we switch this to the non-shortened URL? I know short links from some services expire after a bit. Here's the full link: https://archive.org/details/djangocon-us-2017-django-vs-flask

**Comment by kojoidrissa** [comment]
@SaraDGore GREAT job adding all these. The build seems to be failing due to a 404 error with `http://aaspokane.org/aa-meeting-schedules`. But when I paste that link into my browser, it works. Not sure what's going on. :-(

**Comment by SaraDGore** [comment]
The recovery link also works for me. I went and changed the shortened link to a real one.

**Comment by KatherineMichel** [comment]
I added my slides and ran it locally. The slides link is top center. Is this the intended design?  
slides-screenshot

**Comment by jefftriplett** [comment]
@SaraDGore great work! Thank you so much.

@KatherineMichel yes, once we add the youtube video links, it evens out.

**Comment by jefftriplett** [comment]
@kojoidrissa the AA link bug is fixed in: #298

**Pull request merged**